backend/app/controllers/spree/admin/users_controller.rb

```diff
@@ -107,56 +117,43 @@ def collection
         end
 
       private
-
-      def set_roles
-        if @roles_ids
-          @user.spree_roles = Spree::Role.where(id: @roles_ids)
+        def user_params
+          params.require(:user).permit(PermittedAttributes.user_attributes |
+                                       [:spree_role_ids,
+                                        ship_address_attributes: PermittedAttributes.address_attributes,
+                                        bill_address_attributes: PermittedAttributes.address_attributes])
         end
-      end
 
-      def extract_roles_from_params
-        if params[:user]
-          @roles_ids = params[:user].delete("spree_role_ids")
+        # handling raise from Spree::Admin::ResourceController#destroy
+        def user_destroy_with_orders_error
+          invoke_callbacks(:destroy, :fails)
+          render :status => :forbidden, :text => Spree.t(:error_user_destroy_with_orders)
         end
-      end
 
-      def user_params
-        params.require(:user).permit(PermittedAttributes.user_attributes |
-                                     [:spree_role_ids,
-                                      ship_address_attributes: PermittedAttributes.address_attributes,
-                                      bill_address_attributes: PermittedAttributes.address_attributes])
-      end
-
-      # handling raise from Spree::Admin::ResourceController#destroy
-      def user_destroy_with_orders_error
-        invoke_callbacks(:destroy, :fails)
-        render status: :forbidden, text: Spree.t(:error_user_destroy_with_orders)
-      end
-
-      # Allow different formats of json data to suit different ajax calls
-      def json_data
-        json_format = params[:json_format] || 'default'
-        case json_format
-        when 'basic'
-          collection.map { |u| { 'id' => u.id, 'name' => u.email } }.to_json
-        else
-          address_fields = [:firstname, :lastname, :address1, :address2, :city, :zipcode, :phone, :state_name, :state_id, :country_id]
-          includes = { only: address_fields, include: { state: { only: :name }, country: { only: :name } } }
+        # Allow different formats of json data to suit different ajax calls
+        def json_data
+          json_format = params[:json_format] or 'default'
+          case json_format
+          when 'basic'
+            collection.map { |u| { 'id' => u.id, 'name' => u.email } }.to_json
+          else
+            address_fields = [:firstname, :lastname, :address1, :address2, :city, :zipcode, :phone, :state_name, :state_id, :country_id]
+            includes = { :only => address_fields , :include => { :state => { :only => :name }, :country => { :only => :name } } }
```

Comment:
Line is too long. [129/100]<br>Use the new Ruby 1.9 hash syntax.<br>Space found before comma.